install_files/ansible-base/roles/common/templates/80securedrop.j2

```diff
@@ -1,8 +1,7 @@
 {% if ansible_distribution_release == "focal" %}
-// If automatic reboot is enabled and needed, reboot at the specific
-// time instead of immediately
-//  Default: "now"
-Unattended-Upgrade::Automatic-Reboot-Time "{{ daily_reboot_time }}:00";
+// Reboot should happen after nightly upgrades. Timing of upgrade
+// is configured via apt.daily.timer
+Unattended-Upgrade::Automatic-Reboot-Time "now";
```

Comment:
Warning: my understanding of this option is that running `sudo unattended-upgrades -d` interactively will reboot the system out from under you once updates are complete. This is because we set Unattended-Upgrade::Automatic-Reboot-WithUsers=true. I haven't observed that behavior myself yet, but warning others because it could be surprising.

Reply:
Thanks for calling this out. If we can confirm this behavior & the PR lands, I'll add a small note the docs to this effect.

Reply:
Even though I can see the timers getting fired, the systems did not reboot.

Reply:
To guarantee a reboot, we'd have to update the reboot-flag to fire perhaps hourly. Right now, it fires once every 12 hours. So it's likely that when the update ran, `/var/run/reboot-required` did not (yet) exist.